neonvm/config/kustomization.yaml

```diff
@@ -13,41 +13,23 @@ namePrefix: neonvm-
 #  someName: someValue
 
 bases:
-- ../common/namespace
-- ../common/crd
-- ../common/rbac
-- ../common/controller
-# [WEBHOOK] To enable webhook, uncomment all the sections with [WEBHOOK] prefix including the one in
-# crd/kustomization.yaml
-- ../common/webhook
-# [CERTMANAGER] To enable cert-manager, uncomment all sections with 'CERTMANAGER'. 'WEBHOOK' components are required.
-- ../common/certmanager
+- namespace
+- crd
+- rbac
+- controller
+- webhook
+- certmanager
 # [PROMETHEUS] To enable prometheus monitor, uncomment all sections with 'PROMETHEUS'.
 #- ../common/prometheus
 - vxlan-controller
 - network
-- ../common/device-plugin
+- device-plugin
 
 patchesStrategicMerge:
-- manager_config_patch.yaml
-
-# Protect the /metrics endpoint by putting it behind auth.
-# If you want your controller to expose the /metrics
-# endpoint w/o any authn/z, please comment the following line.
-- manager_auth_proxy_patch.yaml
-
-# [WEBHOOK] To enable webhook, uncomment all the sections with [WEBHOOK] prefix including the one in
-# crd/kustomization.yaml
-- manager_webhook_patch.yaml
-
-# [CERTMANAGER] To enable cert-manager, uncomment all sections with 'CERTMANAGER'.
```

Comment:
No need to keep comments here, above and below?

Reply:
My thought was no, mostly because these comments (similar to #778) are directed at the code author for customizing the output of skaffold